internal/router/router.go

```diff
@@ -33,12 +33,16 @@ func ConfiguredRouter(
 		log.Fatal().Err(err).Msg("creating instrumented user store")
 	}
 
-	svc := getTablelandService(parser, instrUserStore, chainStacks)
+	mesaService := impl.NewTablelandMesa(parser, instrUserStore, chainStacks)
+	userController := controllers.NewUserController(mesaService)
```

Comment:
Use the uninstrumented first to create `userController`, and then instrument it and build `mesaService`.